VisualizationBase/src/Scene.cpp

```diff
@@ -532,6 +532,7 @@ void Scene::setMainViewScalingFactor(qreal factor)
 // Reimplemented in order to detect mouse clicks
 void Scene::mouseMoveEvent(QGraphicsSceneMouseEvent* mouseEvent)
 {
+	lastMousePosition_ = mouseEvent->scenePos();
```

Comment:
I introduced it for playing around with the next step - to show the switching menu for view items. Currently it is not used actually.